coverage_config_x86_64.json

```diff
@@ -1,5 +1,5 @@
 {
-  "coverage_score": 100,
+  "coverage_score": 94.5,
```

Comment:
Oh noes

Reply:
:broken_heart:

Reply:
It's the Display thing... :( And we don't like coverage tests :(.